codalab/server/bundle_manager.py

```diff
@@ -28,6 +28,8 @@
 # When using the REST api, it is allowed to set Memory to 0 but that means the container has unbounded
 # access to the host machine's memory, which we have decided to not allow
 MINIMUM_REQUEST_MEMORY_BYTES = 4 * 1024 * 1024
+# Offset max user disk quota bytes to 0.5G when computing the default amount of disk space to request
```

Comment:
Let's call it "slack" instead of "offset" to convey more of the meaning.  Update the comment to explain this better (The default disk request is disk quota left - slack.)

Reply:
Updated!